README.md

```diff
@@ -1 +1,132 @@
-# Final-capstone-project
\ No newline at end of file
+# Final-capstone-project
+
+# Power Spokes
+
+Power-Spokes is a booking app for motorcycles where users get to book a reservation for a motorcycles
+
+You can see the backend for this project(https://github.com/Addisu87/capstone-appointment-back-end).
+
+## Kanban Board
+
+[Kanban Board Link]()
+
+- We are four (3) in this team.
+
+  - @Emmykage
+  - @Isaacpitwa
+  - @Addisu87
+
+### Our Kanban board
+
+https://github.com/users/Emmykage/projects/4/views/1
+
+## Built With
+
+- React
+- Redux
+- Deployed with Netlify
+
+## Live Demo & Screenshots
+
+> [POWER-STROKE](https://final-capstone-project-udqc.vercel.app/)
+
+<table>
+<tr>
+<td><img src="src/img/splash.png" alt="splash page"></td>
+<td><img src="src/img/signup.png" alt="signup page"></td>
+<td><img src="src/img/login.png" alt="login page"></td>
```

Comment:
- [ ] The file `App.test.js` is the basic test that came with React.  Kindly delete this file if you are not using it 💪